spot/plugins/SkyCam.py

```diff
@@ -170,40 +298,94 @@ def update_image(self, imgpath):
         else:
             img = RGBImage(logger=self.logger)
         img.load_file(imgpath)
-
-        # rotate and transform image as necessary to align with plot
-        # orientation
-        data_np = img.get_data()
-        data_np = trcalc.transform(data_np, flip_x=flip_x,
-                                   flip_y=flip_y, swap_xy=swap_xy)
-        if not np.isclose(rot_deg, 0.0):
-            data_np = trcalc.rotate_clip(data_np, rot_deg,
-                                         rotctr_x=cx, rotctr_y=cy)
+        self.logger.info(imgpath)
```

Comment:
Suggest adding a more meaningful log message, like:
```python
self.logger.info(f"image to be loaded is: {imgpath}")
```
also, this message would be more useful if moved to the top of the function.